lab-claudia/app/entry.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+// require webpack assets
+require('./scss/main.scss');
+
+// npm modules
+const cowsay = require('cowsay-browser');
+const angular = require('angular');
+
+// app modules
+
+// angular module
+// creates an angular module called demoA (has 2 arguments - setter)
+const demoApp = angular.module('demoApp', []);
+//demoApp is the entry point to the app
+//ng-app is demo app- route module is bound to this tag
+//angular can only be done inside this tag
+
+// angular constructor - all controllers are constructors
+// services can be injected into controller
+// services are helper functions - lib directory - supporting code
+demoApp.controller('CowsayController', [ '$log', CowsayController]);
+
+function CowsayController($log){
+  $log.debug('init cowsayCtrl');
+  this.title = 'Derp cows';
+  this.data = [];
+
+  cowsay.list((err, cowfiles) => {
```

Comment:
@loomnugget these could be pulled out of the constructor and added as static methods on the controller, for readability